specification/purview/data-plane/Azure.Analytics.Purview.PDS/preview/2021-01-01-preview/pds.json

```diff
@@ -0,0 +1,899 @@
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/PolicyEventsResponse"
+            }
+          },
+          "304": {
+            "description": "Not Modified"
+          },
+          "400": {
+            "description": "Bad Request",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponseModel"
+            }
+          },
+          "404": {
+            "description": "Not Found",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponseModel"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "PolicyEvents3_Get": {
+            "$ref": "./examples/PolicyEvents3_Get.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "AccountModelBase": {
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string"
+        },
+        "name": {
+          "type": "string"
+        },
+        "location": {
+          "type": "string"
+        },
+        "tenantId": {
+          "type": "string"
+        },
+        "gatewayEndpoint": {
+          "type": "string"
+        }
+      }
+    },
+    "Consent": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "boolean"
+        },
+        "properties": {
+          "$ref": "#/definitions/Properties"
+        },
+        "excludedScopes": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "DatasourcePreferencesResponse": {
+      "type": "object",
+      "properties": {
+        "consent": {
+          "$ref": "#/definitions/Consent"
+        },
+        "accountInfo": {
+          "$ref": "#/definitions/AccountModelBase"
+        }
+      }
+    },
+    "ErrorModel": {
+      "type": "object",
+      "properties": {
+        "code": {
+          "type": "string"
+        },
+        "message": {
+          "type": "string"
+        },
+        "target": {
+          "type": "string"
+        },
+        "details": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ErrorModel"
+          }
+        }
+      }
+    },
+    "ErrorResponseModel": {
+      "type": "object",
+      "properties": {
+        "error": {
+          "$ref": "#/definitions/ErrorModel"
+        }
+      }
+    },
+    "PolicyElement": {
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string"
+        },
+        "scopes": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "kind": {
+          "type": "string"
+        },
+        "updatedAt": {
+          "type": "string"
+        },
+        "version": {
+          "format": "int64",
+          "type": "integer"
+        },
+        "elementJson": {
+          "type": "string"
+        }
```

Comment:
This property is odd. From the name I would have thought it was a JSON object, but it is `type: string`. Does that mean it is a serialized JSON object?